snowplow/iglu-client-embedded/schemas/com.metabase/embedded_analytics_js/jsonschema/1-0-1

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
Try to diff this with `1-0-0` locally. But here's the screenshots that capture all the differences.

<img width="2425" height="720" alt="Image" src="https://github.com/user-attachments/assets/d2ea60de-68dd-45e2-b4e0-05c312ae08b8" />

<img width="2421" height="649" alt="Image" src="https://github.com/user-attachments/assets/026b1152-3b5c-4635-bc04-16be99759caa" />